Formula/zyre.rb

```diff
@@ -0,0 +1,61 @@
+# from https://github.com/Homebrew/homebrew-core/pull/6022/files
+class Zyre < Formula
+  desc "Local Area Clustering for Peer-to-Peer Applications"
+  homepage "https://github.com/zeromq/zyre"
+  url "https://github.com/zeromq/zyre/releases/download/v2.0.0/zyre-2.0.0.tar.gz"
+  sha256 "8735bdf11ad9bcdccd4c4fd05cebfbbaea8511e21376bc7ad22f3cbbc038e263"
+
+  head do
+    url "https://github.com/zeromq/zyre.git"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+  end
+
+  option "with-drafts", "Build and install draft classes and methods"
```

Comment:
np. was trying to not get too far into the weeds, the zmq community has a bit of a "merge first ask questions later" sort of culture to it. draft APIs was a way to enable "merge faster" while at the same time provide some stability while the APIs got tested, sorted, etc. good stuff floats to stable and bad stuff eventually gets ripped out.

just a different way to do design, very specific to that culture. a "production" release is triggered when enough people clamour that "This DRAFT is correct, we're using it in production and we need it to be moved to STABLE". if it never gets that much traction, it eventually just gets removed. but a way to thwart usage by those who may not quite understand the difference in their early adoption.